src/app/test_executive/send_payment_test.ml

```diff
@@ -0,0 +1,40 @@
+open Async
+open Integration_test_lib
+
+module Make (Engine : Engine_intf) = struct
+  open Engine
+
+  type network = Network.t
+
+  type log_engine = Log_engine.t
+
+  let config =
+    let open Test_config in
+    { default with
+      block_producers= [{balance= "4000"}; {balance= "3000"}]
+    ; num_snark_workers= 0 }
+
+  let run network log_engine =
+    let open Deferred.Or_error.Let_syntax in
+    let block_producer = Caml.List.nth network.Network.block_producers 0 in
+    let%bind () = Log_engine.wait_for_init block_producer log_engine in
+    (* namespace for this particular testnet *)
+    let namespace = network.Network.namespace in
```

Comment:
This is also a leak. Namespace is specific to kubernetes, and shouldn't be accessible here (nor should it be necessary to access it).

Reply:
Gone now.